test-dsl/pom.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>com.smartbear.readyapi</groupId>
+    <artifactId>test-dsl</artifactId>
+    <version>2.0.0-SNAPSHOT</version>
```

Comment:
groupId and version should be removed from here and parent should be added rather:
``` 
    <parent>
        <artifactId>readyapi4j</artifactId>
        <groupId>com.smartbear.readyapi</groupId>
        <version>2.0.0-SNAPSHOT</version>
    </parent>
```